CONTRIBUTING.rst

```diff
@@ -522,11 +522,30 @@ this:
 
 ::
 
-   python -m nox -e "pytest-3.7(coverage=False)" -- tests/unit/cli/test_batch.py; espeak "Tests done, woohoo!"
+    python -m nox -e "test-3(coverage=False)" -- tests/unit/cli/test_batch.py; espeak "Tests done, woohoo!"
 
 That way you don't have to keep monitoring the actual test run.
 
 
+::
+
+    python -m nox -e "test-3(coverage=False)" -- --core-tests
+
+You can enable or disable test groups locally by passing there respected flag:
+ * "--no-fast-tests" - Tests that are ~10s or faster.
+    Fast tests make up ~75% of tests and can run in 10 to 20 minutes.
+ * "--slow-tests" - Tests that are ~10s or slower.
+ * "--core-tests" - Tests of any speed that test the root parts of salt.
+ * "--flaky-jail" - Test that need to be temporarily skipped.
+
+In Your PR you can enable or disable test groups by setting a label.
+All thought the fast, slow and core tests specified in the change file will always run.
+ * 'test:no-fast"
+ * "test:core"
+ * "test:slow"
+ * "test:flaky-jail"
```

Comment:
When I view the rendered rsT for this bulleted list, the formatting is off. It shows the block-quote formatting in front of this bulleted list, which means there's an indentation issue. You might need to check how many spaces you've got in front of the bullet points.